extensions/guacamole-auth-ldap/src/main/java/org/apache/guacamole/auth/ldap/connection/ConnectionService.java

```diff
@@ -257,7 +260,10 @@ private String getConnectionSearchFilter(String userDN,
             LDAPSearchResults userRoleGroupResults = ldapConnection.search(
                 groupBaseDN,
                 LDAPConnection.SCOPE_SUB,
-                "(&(!(objectClass=guacConfigGroup))(member=" + escapingService.escapeLDAPSearchFilter(userDN) + "))",
+                "(&(!(objectClass=guacConfigGroup))(" 
+                        + confService.getMemberAttribute() 
```

Comment:
This will need to be escaped with `escapingService.escapeLDAPSearchFilter()`, as well.

Reply:
Escaped.